src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/CarouselViewUITests.PositionVisibility.cs

```diff
@@ -1,4 +1,5 @@
-﻿using NUnit.Framework;
+﻿#if ANDROID
```

Comment:
For this one (and other #if statements) should we consider not adding this? Ideally tests run for all platforms unless there is something REALLY specific which I don't think should be a lot of tests. What if we just enable them, will they run fine on other platforms as well and suddenly get a lot of extra tests "for free"?

Reply:
Maybe if we don't want to fix that right now then let's make sure there is an issue for it to go over these and test that

Reply:
Agree, but we need to review every case. I mean, there are many cases that just was validating something in one specific platform and the test only have the reference snapshot for that platform. We just need to generate the rest and validate it.

I can do that in this PR if we want, or create another issue an review it there.